hi-ml-multimodal/src/health_multimodal/image/model/resnet.py

```diff
@@ -31,7 +41,10 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         x3 = self.layer3(x2)
         x4 = self.layer4(x3)
 
-        return x4
+        if return_skip:
```

Comment:
Maybe a simple unit test for this would be helpful:

Input (128x128x3) random tensor -> making sure that the output shape is consistent, e.g. 64x64xn1_features, ... 32x32xn2_features

Many thanks for creating this PR.

Reply:
Thanks for this Mark!